libs/langchain/langchain/llms/watsonx.py

```diff
@@ -0,0 +1,80 @@
+from typing import Any, Dict, List, Optional
+
+from langchain.callbacks.manager import (
+    Callbacks,
+)
+from langchain.llms.base import LLM
+from langchain.pydantic_v1 import Field
+from langchain.utils import get_from_env
+
+
+def _get_api_key() -> str:
+    return get_from_env("api_key", "WATSONX_API_KEY")
+
+
+class Watsonx(LLM):
+    """WatsonX LLM wrapper."""
+
+    model_name: str = "tiiuae/falcon-40b"
```

Comment:
Would you be able to document the parameters? Especially ones that might not be obvious for interpretation?

Here's a good example of the format:

https://github.com/langchain-ai/langchain/blob/01e9d7902d4bdd0de88bc8d63c96d8d8894b8c4c/libs/langchain/langchain/llms/anthropic.py#L27-L27